app/services/concerns/course/user_invitation_service/parse_invitation_concern.rb

```diff
@@ -58,6 +58,20 @@ def partition_unique_users(users)
     [unique_users.values, duplicate_users]
   end
 
+  # Remove invitees with roles higher than that of inviter.
+  # Currently our course user roles are not ranked, so invitation's role are restricted
+  # such that TAs can only invite students.
+  #
+  # @param [Array<Hash>] users
+  # @return [Array<Hash>] users
+  def restrict_invitee_role(users)
+    if @current_user.course_users.where(course: @current_course).first.role == 'teaching_assistant'
+      users.select { |user| user[:role].to_s == 'student' }
```

Comment:
Change all the users to student role instead. So they still get invited, but only as students.